kamranahmedse/developer-roadmap
# Fix React Native PDF download issue

# Fix React Native Roadmap PDF Content

## Description

This pull request addresses the issue where the React Native roadmap PDF (`public/pdfs/roadmap/react-native.pdf`) incorrectly included content from the C++ roadmap before the React Native roadmap.

## Changes Made

- Removed the C++ roadmap content from the React Native PDF.
- Verified and ensured that the corrected PDF now only contains the React Native roadmap content.

## Context

The issue occurred when the download button for the React Native roadmap PDF resulted in a file that started with C++ roadmap content before transitioning to the correct React Native content. This fix removes the extraneous C++ content, providing users with a seamless and accurate React Native roadmap experience.

## How to Verify

1. Download the React Native roadmap PDF from the specified location (`public/pdfs/roadmap/react-native.pdf`).
2. Open the downloaded PDF and confirm that it now only contains the React Native roadmap content.

## Additional Notes

- No changes were made to other files or directories.

## Related Issue

Closes #4795
